Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -7074,6 +7088,15 @@ void WebPageProxy::decidePolicyForNavigationAction(Ref<WebProcessProxy>&& proces
         sendCachedLinkDecorationFilteringData();
 #endif
 
+    transaction = std::nullopt;
```

Comment:
This is clearly not ratified in the style guide, but I think we're leaning towards `{ }` instead of `std::nullopt` nowadays.

Reply:
That's definitely debatable.  832 vs 1362 instances of each.